reporter/src/main/resources/templates/asciidoc/disclosure_document.ftl

```diff
@@ -31,7 +31,12 @@ Excluded projects and packages are ignored.
 :sectnums:
 :toc: preamble
 
-= Disclosure Document
+[#assign errorTitle = "DISCLAIMER! THERE ARE UNRESOLVED ISSUES IN THE ORT RESULT.
+    THIS DOCUMENT SHOULD NOT BE USED AS A DISCLOSURE DOCUMENT UNTIL THESE ISSUES ARE RESOLVED"?replace("\n", " ")]
+
+[#--The alert role needs to be defined in the pdf-theme file, where the color can be customized.--]
+[#--If not present, the text is diplayed normally.--]
+= [#if helper.hasUnresolvedIssues()][.alert]#${errorTitle}#[#else] Disclosure Document[/#if]
```

Comment:
What does the leading `= ` do here?

Reply:
The number of `=` characters refers to the header level. Similar to `#` in markdown.
In this case, this marks the document title.